pinot-controller/src/main/resources/app/components/SimpleAccordion.tsx

```diff
@@ -120,13 +145,32 @@ export default function SimpleAccordion({
         }
       </AccordionSummary>
       <AccordionDetails className={clsx(classes.details, detailsContainerClass)}>
-        {showSearchBox ?
-          <SearchBar
-            // searchOnRight={true}
-            value={searchValue}
-            onChange={(e) => handleSearch(e.target.value)}
-          />
-          : null}
+        {needsControlsContainer ? (
+          // New layout: search + additional controls in container
+          <div className={classes.controlsContainer}>
+            {showSearchBox && (
+              <div className={classes.searchBarContainer}>
```

Comment:
I don't think we are using this anywhere
we are using additionalControls for Table and not accordion.